integration/spark/src/main/java/org/apache/carbondata/spark/load/CarbonLoaderUtil.java

```diff
@@ -953,66 +959,6 @@ public static void checkAndCreateCarbonDataLocation(String carbonStorePath, Stri
   }
 
   /**
-   * method to distribute the blocklets of a block in multiple blocks
```

Comment:
May be we should take a call for removing blocklet distribution. For filter queries with small number of blocks to scan it is very helpful to process faster.